security_monkey/alerters/slack_alerter.py

```diff
@@ -0,0 +1,38 @@
+"""alert slack with any auditor"""
+
+import json
+
+import requests
+from security_monkey import app
+
+
+def alert_slack(auditor):
```

Comment:
This should be a subclass of the `AlerterType` https://github.com/Netflix/security_monkey/blob/develop/security_monkey/alerters/custom_alerter.py

This code should be implemented within the constructor for the class.